src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.csproj

```diff
@@ -37,9 +37,18 @@
   <Target Name="WriteFullFrameworkResolverManifest" AfterTargets="AfterBuild" Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <PropertyGroup>
       <_ResolverManifestDir>$(OutputPath)\SdkResolvers\$(AssemblyName)\</_ResolverManifestDir>
-      <_TargetPathRelativePath>$([System.IO.Path]::GetRelativePath('$(_ResolverManifestDir)','$(TargetPath)'))</_TargetPathRelativePath>
+      <_TargetPathRelativePath>..\..\$(TargetFileName)</_TargetPathRelativePath>
+    </PropertyGroup>
+
+    <!-- Path.GetRelativePath doesn't exist on full framework. So we hard-code the relative path, and then if we're running on .NET, we check
+         to make sure the value we hard-coded is the same as we would get by calling Path.GetRelativePath -->
+    <PropertyGroup Condition=" '$(MSBuildRuntimeType)' == 'Core'">
+      <_TargetPathRelativePathCheck>$([System.IO.Path]::GetRelativePath('$(_ResolverManifestDir)','$(TargetPath)'))</_TargetPathRelativePathCheck>
     </PropertyGroup>
 
+    <Error Condition=" '$(MSBuildRuntimeType)' == 'Core' And '$(_TargetPathRelativePath)' != '$(_TargetPathRelativePathCheck)'"
```

Comment:
@dsplaisted should this error come from a resource so that it can be localized and do we need to assign an SDK error number to it?

Reply:
Ignore my previous comment, this is our project. For some reason I thought this was in the resolver targets.